src/Standards/Squiz/Docs/WhiteSpace/MemberVarSpacingStandard.xml

```diff
@@ -0,0 +1,88 @@
+<documentation title="Member Var Spacing">
+    <standard>
+    <![CDATA[
+    There should be one blank line before the first property (member variable).
```

Comment:
Just wondering if the phrasing should be made more strict ? (for this one and the second one)
```suggestion
    There should be exactly one blank line before the first property (member variable).
```

Reply:
Updated in e2342396e